### PR TITLE
HCX-86 Use local jar instead of jitpack

### PR DESCRIPTION
**Pull request by NatalieKor**

### CODE REVIEW

File: Dockerfile.deploy [hunk 1]
```diff
@@ -2,11 +2,12 @@ FROM alpine as artifacts
 
 RUN apk add git maven openjdk8 bash zip curl -f
 
-COPY ./demo_processes /usr/src/demo_processes
-COPY ./org            /usr/src/org
-COPY ./src            /usr/src/src
-COPY ./seed           /usr/src/seed
-COPY ./pom.xml        /usr/src/pom.xml
+COPY ./demo_processes/pizza_order/src     /usr/src/demo_processes/pizza_order/src
+COPY ./demo_processes/pizza_order/pom.xml /usr/src/demo_processes/pizza_order/pom.xml
```
Comment (dolfinus) [inline Dockerfile.deploy:6]: You're using build.sh script few lines below but don't copy it into an image. It'll fail.

Comment (NatalieKor) [reply Dockerfile.deploy]: This task was in development, again. Thank you for your attention, cool 😮

Comment (dolfinus) [reply Dockerfile.deploy]: If it is still in development, then add WIP into the pull request name and don't change status in Jira to Awaiting review.

File: Dockerfile.deploy [hunk 2]
```diff
@@ -2,11 +2,15 @@ FROM alpine as artifacts
...
 
-COPY ./demo_processes /usr/src/demo_processes
-COPY ./org            /usr/src/org
-COPY ./src            /usr/src/src
-COPY ./seed           /usr/src/seed
-COPY ./pom.xml        /usr/src/pom.xml
+COPY ./demo_processes/pizza_order/src      /usr/src/demo_processes/pizza_order/src
+COPY ./demo_processes/pizza_order/pom.xml  /usr/src/demo_processes/pizza_order/pom.xml
+COPY ./demo_processes/pizza_order/build.sh /usr/src/demo_processes/pizza_order/build.sh
+COPY ./demo_processes/pizza_order/homs.yml /usr/src/demo_processes/pizza_order/homs.yml
```
Comment (dolfinus) [inline Dockerfile.deploy:8]: This file is not used for building the business process, you can skip it.

File: demo_processes/pizza_order/pom.xml [hunk 3]
```diff
@@ -53,18 +47,19 @@
...
-      <scope>provided</scope>
-    </dependency>
     <dependency>
       <groupId>org.codehaus.groovy</groupId>
       <artifactId>groovy-all</artifactId>
       <version>${groovy.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sample</groupId>
```
Comment (GideonEXE) [inline demo_processes/pizza_order/pom.xml:57]: Hmm... I think it should be something like this "org.camunda-ext"

**Pull request merged**